tests/bsim/bluetooth/ll/bis/testcase.yaml

```diff
@@ -0,0 +1,39 @@
+common:
+  build_only: true
+  bsim_copy_exe: true
+  tags: bluetooth
+
+tests:
+  bluetooth.ll.bis:
```

Comment:
A though: maybe it would be nicer to name these so as to match directly the old name where possible? (to make it easier for users to guess the test name, and reduce a bit the number of bsim_exe_name overrides)

Reply:
I agree, that the names of tests in `testcase.yaml` should correspond to old names as much as possible. Unfortunately, trying to preserve old exe names is almost impossible without using `bsim_exe_name` option. For example, in this particular test `bluetooth.ll.bis`, to make it possible to keep old exe name, I need to change this name literally to `prj.conf` - only this could ensure, that final name of copied exe would be like old one: `bs_<platform_name>_tests_bsim_bluetooth_ll_bis_prj_conf`. Name like `prj.conf` is not meaningful and it does not follow rules of test naming nomenclature in Twister:
https://docs.zephyrproject.org/latest/develop/test/twister.html#test-cases

> 2. Each test identifier shall start with a section followed by a subsection separated by a dot. For example, a test that covers semaphores in the kernel shall start with kernel.semaphore.

To reduce number of used `bsim_exe_name` in the new `testcase.yaml` files it should be considered also renaming of executable files in bsim "test scripts" - but this requires also changes in BabbleSim CI (switching to building by Twister) and I don't want to do that in this PR. So, I decided to use `bsim_exe_name` to maintain backward compatibility and to show how this option can be used in practice.

Reply:
Fair enough.